watson-developer-cloud/discovery-components
# Fix: 204 faster CI Doc tests and more tests

#### What do these changes do/fix?
Unit tests for CI Document no longer take 20 seconds each, all are around 0.5 seconds. There are also more tests implemented to increase our code coverage.

https://zenhub.ibm.com/app/workspaces/discovery-unified-5728bf39d2c20f776e05d0ef/issues/watson-discovery/docviz-squad-issue-tracker/204

#### How do you test/verify these changes?
`yarn test CIDocument` and see that the tests pass.

`Run yarn test:coverage` and view the function and branch coverage of the `CIDocument\utils\enrichmentUtils.ts` is up to statements: 100%, branches: 83.33%, and functions: 100% .

#### Have you documented your changes (if necessary)?
N/A

#### Are there any breaking changes included in this pull request?
No
